addons/binding/org.openhab.binding.ikeatradfri/src/main/java/org/openhab/binding/ikeatradfri/handler/IkeaTradfriGatewayHandler.java

```diff
@@ -134,8 +143,7 @@ public void onLoad(CoapResponse response) {
                         String data = response.getResponseText();
                         logger.debug("COAP GET Successful for: {}", url);
                         future.complete(data);
-                    }
-                    else {
+                    } else {
                         logger.warn("COAP GET Error: {} for {}", response.getCode().toString(), url);
                         future.completeExceptionally(null);
```

Comment:
I would tend to agree with you and fixed it with https://github.com/openhab/openhab2-addons/pull/2200/commits/b41085a18645c7d7b3546c73658dfd560b9a5a27